assets/src/editor/events/tickets/editor-ticket/edit-form/use-ticket-editor-id.js

```diff
@@ -6,12 +6,16 @@ import { isModelEntityOfModel } from '@eventespresso/validators';
 /**
  * @function
  * @param {Object} ticket
+ * @param {string} prefix
  * @return {string} editor id for ticket
  */
-const useTicketEditorId = ( ticket ) => (
-	isModelEntityOfModel( ticket, 'ticket' ) ?
+const useTicketEditorId = ( ticket, prefix = '' ) => {
+	const editorId = isModelEntityOfModel( ticket, 'ticket' ) ?
 		`ticket-editor-${ ticket.id }` :
-		''
-);
+		'';
+	return prefix !== '' && editorId !== '' ?
```

Comment:
Why not simply this?
```JS
return prefix && editorId ?
	`${ prefix }-${ editorId }` :
	editorId;
```

Reply:
that's a good question... 🤔 
ya I dunno what I was thinking  🤷‍♂